src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -187,7 +188,7 @@ public static function createConnection($dsn, array $options = [])
                 }
 
                 try {
-                    @$redis->{$connect}($host, $port, $params['timeout'], (string) $params['persistent_id'], $params['retry_interval'], $params['read_timeout']);
+                    @$redis->{$connect}($host, $port, $params['timeout'], (string) $params['persistent_id'], $params['retry_interval'], $params['read_timeout'], $params['ssl'] ?? null);
```

Comment:
If we trust https://github.com/phpredis/phpredis/issues/1600#issuecomment-776109815
this should be `['stream' => $params['ssl'] ?? []]`
would you be able to confirm?

Reply:
The default value in the C++ code is null, I've had issues locally when providing an empty array here. But looking now it seams to be fine. 

null seams to never make a call to redis_set_socket_options, empty array will call it with no values.

Reply:
went ahead and used an empty array

Reply:
`['stream' => ...]` is not need, despite what the comment says?

Reply:
After a second look, you're correct.

['stream' => $param['ssl'] ?? []] for \Redis::connect(), $param['ssl'] ?? [] for \RedisCluster::__construct